pkg/client/clientset_generated/internalclientset/typed/batch/unversioned/batch_client.go

```diff
@@ -81,9 +82,17 @@ func setConfigDefaults(config *restclient.Config) error {
 	}
 	// TODO: Unconditionally set the config.Version, until we fix the config.
 	//if config.Version == "" {
+	// XXX why is above commented out?
 	copyGroupVersion := g.GroupVersion
 	config.GroupVersion = &copyGroupVersion
 	//}
+	// Do we need something like this:
```

Comment:
@caesarxuchao You had suggested that I set the GroupVersion in the config.  But the config does not have a .Version field, and it overrides the setting in the groupVersion in `setConfigDefaults` at pkg/client/clientset_generated/internalclientset/typed/batch//unversioned/batch_client.go:48

Also, do you know what the "until we fix the config" comment above means?